requirements/docs.txt

```diff
@@ -7,3 +7,4 @@
 sphinx
 sunpy-sphinx-theme
 sphinx-gallery
+sphinx_astropy
```

Comment:
the pip name is `sphinx-astropy` not `sphinx_astropy`.

(yes, yes it's a very confusing convention lol)